frontend/src/components/curation/WaitingToPublish.parts.test.tsx

```
@@ -70,4 +93,49 @@ describe('a held card about a part', () => {
     expect(screen.getByRole('link', { name: 'The Wine Glass' }))
       .toHaveAttribute('href', 'https://www.wikidata.org/wiki/Q782639');
   });
+
+  it('says on the row that a picture and its credit are one answer', async () => {
+    const withPicture = held();
+    withPicture.proposed_parts![0].fields.push(
+      { field: 'image_url', old: 'http://old', new: 'http://new', held: true },
```

ESLint

[error] Using http protocol is insecure. Use https instead.

(sonarjs/no-clear-text-protocols)